src/com/ua/HomeWork/l5/GreaterNumber.java

```diff
@@ -0,0 +1,20 @@
+package com.ua.HomeWork.l5;
+
+import java.util.Scanner;
+
+public class GreaterNumber {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("A = ");
+        int A = scanner.nextInt();
```

Comment:
no capital letters/ should be 'int a'

Reply:
I have corrected the errors. Please check